src/sdk.ts

```diff
@@ -726,7 +726,7 @@ export class OpenSeaSDK {
     const getConsiderationItemsFromSellerFees = (
       fees: Fees
     ): ConsiderationInputItem[] => {
-      const sellerFees = new Map(Object.entries(fees?.sellerFees));
+      const sellerFees = fees?.sellerFees as Map<string, number>;
```

Comment:
did you test this?

Reply:
yep covered in `fees.ts` from a prior pr